.github/workflows/release.yml

```
@@ -718,6 +718,12 @@ jobs:
           pattern: artifacts-*-${{github.run_id}}
           path: /tmp
           merge-multiple: true
+      - name: Download api-augment artifacts
+        id: download-js
+        uses: actions/download-artifact@v4
+        with:
+          name: js-api-augment-${{github.run_id}}/
```

I think the `/` is what got that last run mixed up. You can download the artifacts from the run here in case you didn't know: https://github.com/frequency-chain/frequency/actions/runs/11282097284#artifacts

```suggestion
          name: js-api-augment-${{github.run_id}}
```